google-cloud-storage/src/main/java/com/google/cloud/storage/Bucket.java

```diff
@@ -596,11 +596,17 @@ Builder setEtag(String etag) {
     }
 
     @Override
-    Builder setCreateTime(Long createTime) {
+    public Builder setCreateTime(Long createTime) {
       infoBuilder.setCreateTime(createTime);
       return this;
     }
 
+    @Override
+    public Builder setUpdateTime(Long updateTime) {
+      infoBuilder.setUpdateTime(updateTime);
+      return this;
+    }
+
```

Comment:
these fields should be privated user never ever set those times and it's only for outputs only

Reply:
@pmakani thanks for catching this.